website/source/intro/getting-started/remote.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "intro"
+page_title: "Terraform Remote"
+sidebar_current: "gettingstarted-remote"
+description: |-
+  We've now seen how to build, change, and destroy infrastructure from a local machine. However, you can use Atlas by HashiCorp to run Terraform remotely to version and audit the history of your infrastructure.
+---
+
+# Why Use Terraform Remotely?
+We've now seen how to build, change, and destroy infrastructure
+from a local machine. This is great for testing and development,
+however in production environments it is more responsible to run
+Terraform remotely and store a master Terraform state remotely.
+Otherwise it's possible for multiple different Terraform states
+to be stored on developer machines, which could lead to conflicts.
+Additionally by running Terraform remotely, you can move access
+credentials off of developer machines, release local machines from
+long-running Terraform processes, and store a history of
+infrastructure changes to help with auditing and collaboration.
```

Comment:
I think this doesn't hit upon the biggest benefit: collaboration and working in a multi-team org